lib/stats/DefaultStatsFactoryPlugin.js

```diff
@@ -2022,6 +2022,8 @@ const MODULES_GROUPERS = type => ({
 				getKeys: module => {
 					if (!module.name) return;
 					const resource = parseResource(module.name.split("!").pop()).path;
```

Comment:
`parseResource` should only be executed on a path with query. Data URLs do not have queries so that should not be executed here.